ip_country/src/countries.rs

```diff
@@ -1,329 +1,227 @@
-        Country::new(
-            208,
-            "GS",
-            "South Georgia and the South Sandwich Islands",
-            true
-        ),
-        Country::new(209, "SS", "South Sudan", true),
-        Country::new(210, "ES", "Spain", true),
-        Country::new(211, "LK", "Sri Lanka", true),
-        Country::new(212, "SD", "Sudan", true),
-        Country::new(213, "SR", "Suriname", true),
-        Country::new(214, "SJ", "Svalbard and Jan Mayen", true),
-        Country::new(215, "SE", "Sweden", true),
-        Country::new(216, "CH", "Switzerland", true),
-        Country::new(217, "SY", "Syrian Arab Republic", true),
-        Country::new(218, "TW", "Taiwan", true),
-        Country::new(219, "TJ", "Tajikistan", true),
-        Country::new(220, "TZ", "Tanzania", true),
-        Country::new(221, "TH", "Thailand", true),
-        Country::new(222, "TL", "Timor-Leste", true),
-        Country::new(223, "TG", "Togo", true),
-        Country::new(224, "TK", "Tokelau", true),
-        Country::new(225, "TO", "Tonga", true),
-        Country::new(226, "TT", "Trinidad and Tobago", true),
-        Country::new(227, "TN", "Tunisia", true),
-        Country::new(228, "TR", "Turkiye", true),
-        Country::new(229, "TM", "Turkmenistan", true),
-        Country::new(230, "TC", "Turks and Caicos Islands", true),
-        Country::new(231, "TV", "Tuvalu", true),
-        Country::new(232, "UG", "Uganda", true),
-        Country::new(233, "UA", "Ukraine", true),
-        Country::new(234, "AE", "United Arab Emirates", true),
-        Country::new(235, "GB", "United Kingdom", true),
-        Country::new(236, "UM", "United States Minor Outlying Islands", true),
-        Country::new(237, "US", "United States", true),
-        Country::new(238, "UY", "Uruguay", true),
-        Country::new(239, "UZ", "Uzbekistan", true),
-        Country::new(240, "VU", "Vanuatu", true),
-        Country::new(241, "VE", "Venezuela", false),
-        Country::new(242, "VN", "Vietnam", true),
-        Country::new(243, "VG", "British Virgin Islands", true),
-        Country::new(244, "VI", "US Virgin Islands", true),
-        Country::new(245, "WF", "Wallis and Futuna", true),
-        Country::new(246, "EH", "Western Sahara", true),
-        Country::new(247, "YE", "Yemen", false),
-        Country::new(248, "ZM", "Zambia", true),
-        Country::new(249, "ZW", "Zimbabwe", true),
-        Country::new(250, "XK", "Kosovo", true),
-    ];
-    pub static ref INDEX_BY_ISO3166: HashMap<String, usize> = COUNTRIES
-        .iter()
-        .map(|country| (country.iso3166.clone(), country.index))
-        .collect::<HashMap<String, usize>>();
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub struct Countries {
+    countries: Vec<Country>,
+    index_by_iso3166: HashMap<String, usize>,
 }
 
-impl TryFrom<&str> for Country {
-    type Error = String;
+impl Countries {
+    pub fn old_new(countries: Vec<Country>) -> Self {
```

Comment:
Maybe this can go away. The intention was for it to go away: that's why it's named old_new. If there's something still using it, that thing is probably deprecatable.